front-end/src/store/snackbar-slice.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
I think you should disable some specific rule not all eslint.

Reply:
done